views/backend/settings.php

```diff
@@ -69,15 +75,15 @@
                             <div class="card-content white-text">
                                 <?php
                                 foreach($errors as $error){
-                                    echo $error."<br/>";
+                                    echo $error;
                                 }
                                 ?>
                             </div>
                         </div>
                     <?php
                 }else{
                     $model_user->add_modo($name,$email,$role,$token);
-                    header("Location:index.php?page=settings");
+                    header("Location:/gestion-des-admins-et-modos");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function header() is discouraged](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5349276)